crates/holochain/src/core/ribosome/host_fn/hash.rs

```diff
@@ -19,6 +20,9 @@ pub fn hash(
         HashInput::Header(header) => HashOutput::Header(
             holochain_zome_types::header::HeaderHashed::from_content_sync(header).into_hash(),
         ),
+        HashInput::External(bytes) => HashOutput::External({
+            ExternalHash::from_raw_32(holo_hash::encode::blake2b_256(&bytes))
+        }),
```

Comment:
External hashes can only be blake2b? I was imagining we would want other hash types to be available for interop with other systems?

Reply:
@maackle what kind of interop?

Reply:
see the discussion on the hdk fn